glh_linear/include/glh/glh_linear.h

```diff
@@ -50,7 +50,7 @@ glh_linear.h
 #define GLH_LINEAR_H
 
 #include <memory.h>
-#include <math.h>
+#include <cmath>
```

Comment:
We're interested in maintaining only one version of this codebase so here we replicate some work (in spirit, not a cherry-pick) that was done in the other (private?) repository but has not yet been ported to this one.